### ISSUE TITLE
Vector Art in Summary junk double

### ISSUE THREAD

**Issue by nbanyan** [Free-Cities/Free-Cities#952]
I'm not sure if my dairy workers are getting corrupted, but I've had this issue with random girls for a few versions now (in 10.3, 10.4 plus, and loading a 10.4 save into 10.5). It usually happens if they've been under therapy for a while. (attached is a save from a v10.4 new game+)
![fc bug vecter](https://user-images.githubusercontent.com/2677727/29101347-923eeb22-7c7f-11e7-9491-5e464f5539ca.png)
[free-cities-20170808-211844.zip](https://github.com/Free-Cities/Free-Cities/files/1210016/free-cities-20170808-211844.zip)

**Comment by lowercase-donkey** [comment]
http://i.imgur.com/c4jt321.png

**Comment by prndev** [comment]
I fail to reproduce this behaviour in a04da1d.

**Comment by freecitiesdev** [comment]
I _think_ I've seen this before, but I've had trouble reproducing it as well. Does a quick F5 to reload the page resolve this when it happens? That's been my limited experience. Odd.

**Comment by nbanyan** [comment]
No, it's rather persistent even when reloading the save file. In Firefox, Opera and Chrome; it doesn't matter which browser either.

**Comment by nbanyan** [comment]
I found the issue. The vector download pack has a bad svg. The "body/brown/flaccid 6.svg" file is a full body, thin white with chastity cage instead of just an enlarged member. It's this way in all the packs listed in the blog (10.3 and 10.4).

**Comment by lowercase-donkey** [comment]
That helps. Vector pack is depreciated at this point.  Try running the game with vector images enabled, but without the old standalone vector pack download.  Maybe that will get you going.

**Comment by prndev** [comment]
This issue should be closed due to lack of response.
